x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/actions_log_users_filter.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useCallback, useState, useEffect } from 'react';
+import { EuiFieldSearch, EuiFormControlLayout } from '@elastic/eui';
+import { useTestIdGenerator } from '../../../hooks/use_test_id_generator';
+import { FILTER_NAMES } from '../translations';
+import { useActionHistoryUrlParams } from './use_action_history_url_params';
+
+export const ActionsLogUsersFilter = memo(
+  ({
+    isFlyout,
+    onChangeUsersFilter,
+  }: {
+    isFlyout: boolean;
+    onChangeUsersFilter: (selectedUserIds: string[]) => void;
+  }) => {
+    const getTestId = useTestIdGenerator('response-actions-list');
+    const { users: usersFromUrlParams, setUrlUsersFilters } = useActionHistoryUrlParams();
+    const [searchValue, setSearchValue] = useState('');
```

Comment:
should we use the url param as initial value for this? If user reloads the page, will the input be prepopulated with url values?